services/civic-2018-service/service.yaml

```diff
@@ -0,0 +1,148 @@
+Description: >
+    ECS Service - HackOregon Budget API
+Parameters:
+
+    VPC:
+        Description: The VPC that the ECS cluster is deployed to
+        Type: AWS::EC2::VPC::Id
+
+    Cluster:
+        Description: Please provide the ECS Cluster ID that this service should run on
+        Type: String
+
+    DesiredCount:
+        Description: How many instances of this task should we run across our cluster?
+        Type: Number
+        Default: 2
+
+    Listener:
+        Description: The Application Load Balancer listener to register with
+        Type: String
+
+    ConfigBucket:
+        Description: The configuration bucket we want to use
+        Type: String
+
+    DeployTarget:
+        Description: Which environment (Valid Values - integration, production)
+        Type: String
+
+    ProjSettingsDir:
+        Description: the relative directory path where we keep the app config
+        Type: String
+
+    Path:
+        Description: The path to register with the Application Load Balancer
+        Type: String
+        Default: /2018*
+
+Resources:
+
+    Service:
+        Type: AWS::ECS::Service
+        DependsOn: ListenerRule
+        Properties:
+            Cluster: !Ref Cluster
+            Role: !Ref ServiceRole
+            DesiredCount: !Ref DesiredCount
+            TaskDefinition: !Ref TaskDefinition
+            LoadBalancers:
+                - ContainerName: "civic-2018-service"
+                  ContainerPort: 3000
+                  TargetGroupArn: !Ref TargetGroup
+
+    TaskDefinition:
+        Type: AWS::ECS::TaskDefinition
+        Properties:
+            Family: civic-2018-service
+            ContainerDefinitions:
+                - Name: civic-2018-service
+                  Essential: true
+                  Image: 845828040396.dkr.ecr.us-west-2.amazonaws.com/production/civic-2017
```

Comment:
```
Image: 845828040396.dkr.ecr.us-west-2.amazonaws.com/production/civic-2017
```
Why did you use `civic-2017` image for /services/civic-2018-service/service.yaml`?